scripts/sample-data.js

```diff
@@ -30,10 +30,18 @@ const NAMES = [
   '계룡/鷄龍',
```

Comment:
While this PR is still open can you also change the line 16 (can not comment up there :) ) to: `docker exec -ti origin-js node scripts/sample-data.js 42 17`